packages/frontend/src/components/common/Filters/FilterInputs/FilterMultiStringInput.tsx

```diff
@@ -1,31 +1,22 @@
-import { type FilterableItem } from '@lightdash/common';
 import { Group, MultiSelect, Text, type MultiSelectProps } from '@mantine/core';
 import { IconPlus } from '@tabler/icons-react';
 import uniq from 'lodash/uniq';
 import { useCallback, useMemo, useState, type FC } from 'react';
 import MantineIcon from '../../MantineIcon';
-import { useFiltersContext } from '../FiltersProvider';
 import MultiValuePastePopover from './MultiValuePastePopover';
 
 type Props = Omit<MultiSelectProps, 'data' | 'onChange'> & {
-    field: FilterableItem;
```

Comment:
is this component being used somewhere else?

Reply:
Yes this component is being used for the inputs in the current explore, the field prop wasn't being used at all so I could just remove it and reuse the component